db/migrate/20221018121259_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Your user reference should refer to `author`, not `user`. You can use the `to_table` attribute of `foreign_key` to point the database to the user table. Lok at the answer from stack overflow: [https://stackoverflow.com/a/42056089](https://stackoverflow.com/a/42056089)

Reply:
@julie-ify  Thank you for reviewing my project, I will make the necessary corrections as suggested